test/com/m3/octoparts/repository/config/HttpPartConfigRepositorySpec.scala

```diff
@@ -189,6 +190,7 @@ class HttpPartConfigRepositorySpec extends fixture.FunSpec with DBSuite with Mat
         val Some(threadPoolConfig) = hystrixConfig.threadPoolConfig // will fail if not eager loaded
         threadPoolConfig.threadPoolKey should be("swimmingpool")
         threadPoolConfig.coreSize should be(10)
+        threadPoolConfig.queueSize should be(500)
```

Comment:
Perhaps a test for `queueSize` 0 since it seems to be tested for by the DB. What will that error look like, DB error?